From: Maxime Ripard <maxime@cerno.tech>
Date: Tue, 28 Jan 2020 09:37:06 +0100
Subject: ARM: dts: bcm2711: Add HDMI DVP

Now that we have a driver for the DVP, let's add its DT node.

Signed-off-by: Maxime Ripard <maxime@cerno.tech>
---
 arch/arm/boot/dts/bcm2711-rpi.dtsi | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm/boot/dts/bcm2711-rpi.dtsi b/arch/arm/boot/dts/bcm2711-rpi.dtsi
index 69ef73991921..23373e726b57 100644
--- a/arch/arm/boot/dts/bcm2711-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2711-rpi.dtsi
@@ -5,6 +5,14 @@
 	soc {
 		/delete-node/ v3d@7ec00000;
 		/delete-node/ mailbox@7e00b840;
+
+		dvp: clock@7ef00000 {
+			compatible = "brcm,brcm2711-dvp";
+			reg = <0x7ef00000 0x10>;
+			clocks = <&clk_108MHz>;
+			#clock-cells = <1>;
+			#reset-cells = <1>;
+		};
 	};
 
 	__overrides__ {
@@ -38,6 +46,13 @@
 	     /* Add a label */
 	};
 
+	clk_108MHz: clk-108M {
+		#clock-cells = <0>;
+		compatible = "fixed-clock";
+		clock-frequency = <108000000>;
+		clock-output-names = "108MHz-clock";
+	};
+
 	firmware_clocks: firmware-clocks {
 		compatible = "raspberrypi,firmware-clocks";
 		raspberrypi,firmware = <&firmware>;
